sdk/core/core-https/review/core-https.api.md

```diff
@@ -52,6 +52,9 @@ export class DefaultHttpsClient implements HttpsClient {
     sendRequest(request: PipelineRequest): Promise<PipelineResponse>;
 }
 
+// @public
+export function delay<T>(t: number, value?: T): Promise<T | void>;
```

Comment:
when do we use the value param, anyway?

Reply:
I did a grep through out the repo and could not find any use for it.

Reply:
I think if we're going to add it back into `core-https`, we should ditch the value param

Reply:
Done.